.github/workflows/check-layer.yml

```diff
@@ -13,6 +13,4 @@ jobs:
 
       - name: Run yocto-check-layer
         run: |
-          mkdir -p ../build
-          cd ../build
-          kas shell ../meta-qcom-hwe/ci/qcs6490-rb3gen2-core-kit.yml --command "yocto-check-layer-wrapper `pwd`/../meta-qcom-hwe --dependency `pwd`/poky/meta `pwd`/meta-qcom --no-auto-dependency"
+          meta-qcom-hwe/ci/yocto-check-layer.sh
```

Comment:
Run is already, by default, running from the parent folder (which is this repo), so no need to specify meta-qcom-hwe.

Change to './ci/yocto-check-layer.sh' or similar.

Reply:
yup, was trying to understand where it was executed.
thanks